src/moduler/aktivitet/visning/hjelpekomponenter/CustomAlertstripe.tsx

```diff
@@ -0,0 +1,20 @@
+import { Element } from 'nav-frontend-typografi';
+import React from 'react';
+
+import { ReactComponent as VarselIkon } from '../../../../Ikoner/advarsel-ikon.svg';
+import styles from './CustomAlertstripe.module.less';
+
+interface Props {
+    tekst: string;
+    sectionClassName?: string;
+    textClassName?: string;
+}
+
+export const CustomAlertstripe = (props: Props) => {
+    return (
```

Comment:
Her kan du returnere direkte
```suggestion
export const CustomAlertstripe = (props: Props) =>  (
```